core/setup.py

```diff
@@ -29,8 +29,8 @@
     'requests',
     'matplotlib',
     'paramiko>=2.4',
-    'dask>=2021.09.1',
-    'distributed>=2021.09.1',
+    'dask>=2021.09.1,<=2021.11.2',
```

Comment:
Add comment saying that 2021.12.0 causes ray to error, we will want to avoid capping this forever to this version and check if the newer dask versions will work in future.

Reply:
also mention version of ray and maybe a link to the error we saw